src/main/java/ee/tuleva/onboarding/common/Utils.java

```diff
@@ -0,0 +1,17 @@
+package ee.tuleva.onboarding.common;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+
+public class Utils {
+
+    public static Instant parseInstant(String format) {
+        try {
+            return new SimpleDateFormat("yyyy-MM-dd").parse(format).toInstant();
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
Prints to standard out, better to avoid (not logged to file).
Not sure if it makes sense to catch the `ParseException` and return `Instant.now()` in that case?
I'd rather fail fast instead of silently.

Reply:
If we remove the `catch` block then maybe we won't need the `Utils` class at all - we can just use `new SimpleDateFormat("yyyy-MM-dd").parse(format).toInstant();` inline where needed.